File: erezervisi_api/eRezervisi/eRezervisi.Common.Dtos/User/UpdateSettingsDto.cs
namespace eRezervisi.Common.Dtos.User
{
    public class UpdateSettingsDto 
    {
        public bool ReceiveEmails { get; set; }
        public bool ReceiveNotifications { get; set; }
        public bool MarkObjectAsUncleanAfterReservation { get; set; }
    }
}

File: erezervisi_desktop/erezervisi_desktop/lib/models/requests/user/update_settings_dto.dart
class UpdateSettingsDto {
  final bool receiveEmails;
  final bool receiveNotifications;
  final bool markObjectAsUncleanAfterReservation;

  UpdateSettingsDto(
      {required this.receiveEmails,
      required this.receiveNotifications,
      required this.markObjectAsUncleanAfterReservation});

  Map<String, dynamic> toJson() {
    return {
      'receiveEmails': receiveEmails,
      'receiveNotifications': receiveNotifications,
      'markObjectAsUncleanAfterReservation': markObjectAsUncleanAfterReservation
    };
  }
}
